spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -261,7 +261,8 @@ def _get_size_of_generator_information(self, in_edges):
         """ Get the size of the synaptic expander parameters
         """
         gen_on_machine = False
-        size = 0
+        # Overhead added by _write_synaptic_matrix_and_master_population_table
+        size = 1 * BYTES_PER_WORD
```

Comment:
I think this actually comes from the direct matrix region (region 10).

Reply:
Absolutely the 1 * BYTES_PER_WORD is applied to the Direct matrix region.

It is the method _write_synaptic_matrix_and_master_population_table that add is there.

My assumption was that the method _get_size_of_generator_information was used to calc the size of the Direct matrix region

Reply:
moved.